Add OpenScPCA-nf docs page (#582)
AlexsLemonade/OpenScPCA-analysis: docs/ensuring-repro/openscpca-nf/index.md

# The OpenScPCA Nextflow workflow

> In addition to the `OpenScPCA-analysis` repository, the Data Lab maintains [a separate repository called `OpenScPCA-nf`](https://github.com/AlexsLemonade/OpenScPCA-nf).
> The `OpenScPCA-nf` repository holds a [Nextflow workflow](https://www.nextflow.io/) which we use to generate results from mature `OpenScPCA-analysis` modules.

Reviewer: This felt choppy, so I combined it into one sentence.
```suggestion
In addition to the `OpenScPCA-analysis` repository, the Data Lab maintains a separate [`OpenScPCA-nf` repository](https://github.com/AlexsLemonade/OpenScPCA-nf) that holds a [Nextflow workflow](https://www.nextflow.io/) which we use to generate results from mature `OpenScPCA-analysis` modules.
```

Reviewer: In another place, we used the phrase "official results". I don't love that phrase, but I like the sentiment of it. We should say something like that here?

> Specifically, once an analysis module has been completed, the Data Lab will port the analysis code into the `OpenScPCA-nf` repository and add it to the Nextflow workflow.

Reviewer: I felt like this wasn't really making the previous sentence more specific, but rather adding context, so I removed the first word...
```suggestion
As analysis modules are completed, the Data Lab will port the analysis code into the `OpenScPCA-nf` repository and add it to the Nextflow workflow.
```
